sql/plan/ddl_trigger.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Liquidata, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plan
+
+import (
+	"fmt"
+	"github.com/liquidata-inc/go-mysql-server/sql"
+)
+
+type TriggerOrder struct {
+	PrecedesOrFollows string // PrecedesStr, FollowsStr
```

Comment:
I think we should turn these into "golang enums" rather than relying on the strings. We could do that in the parser.

Reply:
That was my first instinct too. I had go-enums in the parser originally, but it was kind of a pain, so I changed them back to strings. I will definitely sort this out before we make any public interfaces for triggers.